examples/complete/variables.tf

```
@@ -1,4 +1,42 @@
-variable "example" {
+variable "region" {
   type        = string
-  description = "The value which will be passed to the example module"
+  description = "AWS Region"
+}
+variable "create_iam_role" {
+  description = "Flag to indicate whether an IAM Role should be created to grant the proper permissions for AWS Config"
+  type        = bool
+  default     = false
+}
+
+variable "create_sns_topic" {
+  description = <<-DOC
+    Flag to indicate whether an SNS topic should be created for notifications
+    If you want to send findings to a new SNS topic, set this to true and provide a valid configuration for subscribers
+  DOC
+
+  type    = bool
+  default = false
+}
+
+variable "force_destroy" {
+  type        = bool
+  description = "(Optional, Default:false ) A boolean that indicates all objects should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable"
```

```suggestion
  description = "A boolean that indicates all objects should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable"
```

We generate the description whether it's optional or not by `make readme`

---

I will update. FWIW, that was a copy/paste from the [terraform-aws-s3-log-storage](https://github.com/cloudposse/terraform-aws-s3-log-storage/blob/master/variables.tf#L25) module.

---

a81682f2aba880e220ffd343c77acaaa9478a693